corehq/apps/reports/filters/users.py

```diff
@@ -326,40 +301,40 @@ def user_es_query(cls, domain, request):
             else:
                 return q.filter(id_filter)
 
-
     @classmethod
-    @memoized
```

Comment:
ok couldn't get this to work because of the list arg, but i'm going to go ahead and say it's OK because  all the calls to elasticsearch in this function are `memoized` as well. `QuickCache` couldn't pickle the return value and `memoized` and `lru_cache` couldn't hash the list param. could put effort to expanding the functionality but decided it was a bit out of scope